lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -243,7 +242,7 @@ def create_post_for_auth_or_purchase(money, payment, options)
           add_customer_data(post, options)
           add_metadata(post, options)
           post[:description] = options[:description]
-          post[:statement_description] = options[:statement_description]
+          post[:statement_descriptor] = options[:statement_description]
```

Comment:
This is actually the change that precipitated the need for more explicit version handling @girasquid; customers using freshly created Stripe accounts are broken today on Spreedly when passing a statement descriptor due to the floating version requirement. These changes should make this a non-breaking change.